From: Peter Volkov <pva@gentoo.org>
Date: Mon, 3 Mar 2008 09:52:27 +0000 (+0300)
Subject: Fix proc compilation
X-Git-Url: http://git.openvz.org/?p=linux-2.6.24-openvz;a=commitdiff_plain;h=da23c1831f33ce142b38534f8e54f985f651707c;hp=9c10803877cbf58bebe511eafa67fee2c615f6f6

Fix proc compilation

fs/proc/generic.c: In function 'xlate_proc_loc_name':
fs/proc/generic.c:330: error: implicit declaration of function 'get_exec_env'
fs/proc/generic.c:330: error: invalid type argument of '->'
fs/proc/generic.c: In function 'proc_lookup':
fs/proc/generic.c:516: error: implicit declaration of function 've_is_super'
---

diff --git a/fs/proc/generic.c b/fs/proc/generic.c
index ff8484e..2260a20 100644
--- a/fs/proc/generic.c
+++ b/fs/proc/generic.c
@@ -22,6 +22,7 @@
 #include <linux/bitops.h>
 #include <linux/spinlock.h>
 #include <linux/completion.h>
+#include <linux/sched.h>
 #include <asm/uaccess.h>
 
 #include "internal.h"
diff --git a/fs/proc/proc_tty.c b/fs/proc/proc_tty.c
index a5e8cd9..a3da5a0 100644
--- a/fs/proc/proc_tty.c
+++ b/fs/proc/proc_tty.c
@@ -13,6 +13,7 @@
 #include <linux/stat.h>
 #include <linux/tty.h>
 #include <linux/seq_file.h>
+#include <linux/sched.h>
 #include <linux/bitops.h>
 
 static int tty_ldiscs_read_proc(char *page, char **start, off_t off,
